scripts/update_stm32_package.py

```diff
@@ -8,7 +8,9 @@
 # usage : 'python update_stm32_package.py <stm32_serie>'
 
 import os
-import sys; sys.dont_write_bytecode = True    # do not generate pyc file
+import sys
+
+sys.dont_write_bytecode = True  # do not generate pyc file
```

Comment:
Why is this needed?

Reply:
It will merge only script files, not pyc ones